crates/cli/src/wasm_bindgen.rs

```diff
@@ -0,0 +1,179 @@
+    omit_imports: bool,
+    #[arg(
+        long,
+        value_name = "VAR",
+        help = "Set a custom output filename (Without extension. Defaults to crate name)"
+    )]
+    out_name: Option<String>,
+    #[arg(long, help = "Include otherwise-extraneous debug checks in output")]
+    debug: bool,
+    #[arg(long, help = "Don't demangle Rust symbol names")]
+    no_demangle: bool,
+    #[arg(
+        long,
+        value_name = "VAR",
+        help = "Name of the global variable to initialize"
+    )]
+    no_modules_global: Option<String>,
+    #[arg(long, help = "Remove the debugging `name` section of the file")]
+    remove_name_section: bool,
+    #[arg(long, help = "Remove the telemetry `producers` section")]
+    remove_producers_section: bool,
+    #[arg(long, help = "Keep exports synthesized by LLD")]
+    keep_lld_exports: bool,
+    #[arg(long, help = "Keep debug sections in Wasm files")]
+    keep_debug: bool,
+    #[arg(
+        long,
+        value_name = "MODE",
+        help = "Whether or not to use TextEncoder#encodeInto",
+        value_parser = ["test", "always", "never"]
+    )]
+    encode_into: Option<String>,
+    #[arg(
+        long,
+        help = "Don't add WebAssembly fallback imports in generated JavaScript"
+    )]
+    omit_default_module_path: bool,
+    #[arg(
+        long,
+        help = "Split linked modules out into their own files. Recommended if possible.\n\
+                If a bundler is used, it needs to be set up accordingly."
+    )]
+    split_linked_modules: bool,
+    #[arg(
+        long = "experimental-reset-state-function",
+        help = "Generate __wbg_reset_state function for WASM reinitialization (experimental)"
+    )]
+    generate_reset_state: bool,
+    // The options below are deprecated. They're still parsed for backwards compatibility,
+    // but we don't want to show them in `--help` to avoid distracting users.
+    #[arg(long, hide = true)]
+    #[deprecated(note = "implied default, only `--no-typescript` is needed")]
+    typescript: bool,
+    #[arg(long, hide = true, group = "target-group")]
+    #[deprecated(note = "use `Args::target` instead")]
+    nodejs: bool,
+    #[arg(long, hide = true, group = "target-group")]
+    #[deprecated(note = "use `Args::target` instead")]
+    web: bool,
+    #[arg(long, hide = true, group = "target-group")]
+    #[deprecated(note = "use `Args::target` instead")]
+    no_modules: bool,
+    #[arg(long, hide = true)]
+    #[deprecated(note = "runtime-detected")]
+    #[allow(dead_code)]
+    weak_refs: bool,
+    #[arg(long, hide = true)]
+    #[deprecated(note = "automatically inferred from the Wasm features")]
+    reference_types: bool,
+}
+
+pub fn run_cli_with_args<I, T>(args: I) -> anyhow::Result<()>
+where
+    I: IntoIterator<Item = T>,
+    T: Into<OsString> + Clone,
+{
+    let args = Args::try_parse_from(args)?;
+    rmain(&args)
+}
+
+fn rmain(args: &Args) -> Result<(), Error> {
+    env_logger::init();
```

Comment:
`env_logger::init()` needs to be left in the binary. In the library environment most likely logger might be already set up globally, or user might decide not to use it, and `env_logger::init()` panics if called twice.

It's really only meant to be called at the top level.